src/data-model-providers/codegen/CodegenDataModelProvider.cpp

```diff
@@ -159,11 +159,13 @@ std::optional<DataModel::ActionReturnStatus> CodegenDataModelProvider::Invoke(co
                                                                               TLV::TLVReader & input_arguments,
                                                                               CommandHandler * handler)
 {
+    bool clusterRecognizedByCHI = false;
     CommandHandlerInterface * handler_interface =
         CommandHandlerInterfaceRegistry::Instance().GetCommandHandler(request.path.mEndpointId, request.path.mClusterId);
 
     if (handler_interface)
     {
+        clusterRecognizedByCHI = true;
```

Comment:
Unfortunately, no.  You could have a CHI registered as a wildcard but the cluster still only exists on some of the endpoints....

Reply:
I don't think we have this check before, and added the check